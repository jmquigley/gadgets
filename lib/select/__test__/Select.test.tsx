'use strict';

import * as assert from 'assert';
import {shallow} from 'enzyme';
import * as React from 'react';
import {Select} from '../index';

test('Test creation of a Select control', () => {
	const ctl = shallow(<Select className="test-class" />);

	assert(ctl);
	expect(ctl).toMatchSnapshot();
});

// TODO: disabling the select control
// TODO: make the select control invisible
// TODO: test adding/retrieving text example from the Select control
