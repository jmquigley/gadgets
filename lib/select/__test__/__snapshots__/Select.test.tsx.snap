// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test creation of a Select control 1`] = `
<ThemeProvider
  theme={
    Object {
      "backgroundColor": "#ffffff",
      "borderColor": "#c0c0c0",
      "color": "#000000",
      "headerBackgroundColor": "#004358",
      "headerForegroundColor": "#ffffff",
      "headerHoverColor": "#ffe11a",
      "hoverColor": "#c0c0c0",
      "inputBorderColor": "#c0c0c0",
      "tooltipBackgroundColor": "gray",
      "tooltipForegroundColor": "white",
      "transitionDelay": "0.5s",
    }
  }
>
  <Styled(Select)
    className="ui-select test-class"
    disabled={false}
    sizing="normal"
    visible={true}
  />
</ThemeProvider>
`;
